INSTALLATION.rst

```diff
@@ -23,6 +23,14 @@ The C libraries required are:
 As it turns out, though these are fairly common libraries, getting them installed in a
 way that `21cmFAST` understands on various operating systems can be slightly non-trivial.
 
+HPC
+~~~
+These libraries will *usually* be available on a HPC environment by using the
+``module load gsl`` and similar commands. Note that just because they are loaded
```

Comment:
In my experience this may not necessarily be true. FFTW used by 21cmFAST is the float, multiprocessing version which may not always be installed.

Reply:
Yeah, true. I've updated it to say "often" instead of usually, and point out that FFTW has to have the right options.